php/class-migrate.php

```diff
@@ -0,0 +1,205 @@
+<?php
+/**
+ * Customize Snapshot.
+ *
+ * @package CustomizeSnapshots
+ */
+
+namespace CustomizeSnapshots;
+
+/**
+ * Customize Snapshot Migration for 4.7 changeset.
+ *
+ * @package CustomizeSnapshots
+ */
+class Migrate {
+
+	const KEY = 'customize_snapshot_migrate';
+
+	/**
+	 * Plugin instance.
+	 *
+	 * @var bool
+	 */
+	public $compat;
+
+	/**
+	 * Migrate constructor.
+	 */
+	public function __construct() {
+		$this->compat = is_back_compat();
```

Comment:
I don't see the need for the `compat` class variable , it's not being used anywhere.